resources/assets/js/features/public-profiles/styles/driver-profile.scss

```diff
@@ -60,10 +60,10 @@
   padding: 20px;
 }
 
-.card {
+.card-comment {
   position:relative;
 }
-.card>.card-header:after,.card>.card-header:before{
+.card>.comment-header:after,.card>.comment-header:before{
```

Comment:
use sass style for describing child classes
```
.card {
  & > .comment-header {
    &:before,&:after {}
  }
}
```

and etc. rewire all file style structure and use sass features here